code/modules/projectiles/pins.dm

```diff
@@ -373,6 +373,16 @@
 	suit_requirement = /obj/item/clothing/suit/bluetag
 	tagcolor = "blue"
 
+/obj/item/firing_pin/monkey
+	name = "monkeylock firing pin"
+	desc = "This firing pin prevents non-monkies from firing a gun."
+	fail_message = "not a monkey!"
+
+/obj/item/firing_pin/monkey/pin_auth(mob/living/user)
+	if(!ismonkey(user))
```

Comment:
```suggestion
	if(!is_simian(user))
```

Reply:
Bit pointless as I believe Gorillas are inherently unable to fire guns.

Reply:
thats cringe